jbpm-installer/build.xml

```diff
@@ -43,6 +43,7 @@
       <include name="antlr-2.7.7.jar"/>
       <include name="hibernate-jpa-2.0-api*.jar"/>
       <include name="jta-1.1.jar"/>
+	  <include name="jboss-transaction-api*.jar"/>
```

Comment:
for hibernate ddl generation need. jta has been most likely replaced with jboss transactions api in bom as that worked previously. Though me might want to remove jta while merging - it's not going to be back ported to 6.3.x anyway